pandas/core/algorithms.py

```diff
@@ -375,46 +375,60 @@ def unique(values):
     >>> pd.unique(pd.Series([2] + [1] * 5))
     array([2, 1])
 
-    >>> pd.unique(pd.Series([pd.Timestamp('20160101'),
-    ...                     pd.Timestamp('20160101')]))
+    >>> pd.unique(pd.Series([pd.Timestamp("20160101"), pd.Timestamp("20160101")]))
     array(['2016-01-01T00:00:00.000000000'], dtype='datetime64[ns]')
 
-    >>> pd.unique(pd.Series([pd.Timestamp('20160101', tz='US/Eastern'),
-    ...                      pd.Timestamp('20160101', tz='US/Eastern')]))
-    array([Timestamp('2016-01-01 00:00:00-0500', tz='US/Eastern')],
-          dtype=object)
-
-    >>> pd.unique(pd.Index([pd.Timestamp('20160101', tz='US/Eastern'),
-    ...                     pd.Timestamp('20160101', tz='US/Eastern')]))
+    >>> pd.unique(
+    ...     pd.Series(
+    ...         [
+    ...             pd.Timestamp("20160101", tz="US/Eastern"),
+    ...             pd.Timestamp("20160101", tz="US/Eastern"),
+    ...         ]
+    ...     )
+    ... )
+    <DatetimeArray>
+    ['2016-01-01 00:00:00-05:00']
+    Length: 1, dtype: datetime64[ns, US/Eastern]
+
+    >>> pd.unique(
+    ...     pd.Index(
+    ...         [
+    ...             pd.Timestamp("20160101", tz="US/Eastern"),
+    ...             pd.Timestamp("20160101", tz="US/Eastern"),
+    ...         ]
+    ...     )
+    ... )
     DatetimeIndex(['2016-01-01 00:00:00-05:00'],
```

Comment:
Format here looks a bit odd?

Reply:
That's the "Black" formatting.

I personally think it's easier on the eyes, but that's not a deal breaker for me.

Can revert if you think it's best to leave it as it was before

Reply:
Sorry if this was misleading, I was referring to the lines you haven‘t touched, they look out of sync now

Reply:
I didn't managed to get the output lines to follow the "black" formatting styles.

What I could do is to make the output to appear in a single line, instead of multiple by using ```\```. i.e

```python
def foo():
    """
    Examples
    --------------
    >> foo()
    Line1 \
Line 2 \
Line 3 \
    """
```

Will result in the docs as:
```
>>> foo()
Line 1 Line 2 Line 3
```

Reply:
Sounds good to me